### ISSUE TITLE
Can't to see the performance of GPT4-V on DESCRIPTION and ASSESSMENT tasks!

### ISSUE THREAD

**Issue by Luciennnnnnn** [VQAssessment/Q-Bench#3]
Hi, the human level performance of GPT4-V on PERCEPTION task is impressive! Is there any plans to test it on other two tasks?

Best regards,
Xin

**Comment by teowu** [comment]
These will be put into a further plan. 

1. For the DESCRIPTION task, one concern is that the current pipeline using GPT-3.5 to evaluate GPT-4V may not be enough accurate, so we are working on `human_eval` to evaluate it (same for open-source models). Please stay tuned as that may take time.

3. For the ASSESSMENT task, one big problem is that GPT-4V does not support `logits` as outputs, which is the basis of our evaluation setting. Another concern is that given the IQA datasets are large, without API, developing a good strategy for it and further testing will be very difficult and time-consuming. We will take a deep dive into this when the GPT-4V API is out.
